README.md

```diff
@@ -1701,6 +1701,7 @@ Other Style Guides
     const truthyCount = array.filter(Boolean).length;
     ```
 
+<a name="variables--linebreak"></a><a name="13.7"></a>
```

Comment:
The first one of these would be great; but we don't add new numbered links, they're only there for legacy reasons.